rio_tiler/io/cogeo.py

```diff
@@ -112,7 +112,7 @@ def __enter__(self):
         """Support using with Context Managers."""
         self.dataset = self.dataset or rasterio.open(self.filepath)
 
-        self.bounds: Tuple[float, float, float, float] = transform_bounds(
+        self.bounds = transform_bounds(
```

Comment:
You could probably define the type for `self.bounds` above with the others?

Reply:
type of self.bounds is defined in the base class